x-pack/plugins/ml/public/application/components/anomalies_table/links_menu.js

```diff
@@ -53,6 +53,51 @@ class LinksMenuUI extends Component {
     };
   }
 
+  openInDiscover = async () => {
+    const {
+      services: { share },
+    } = this.props.kibana;
+    const discoverLocator = share.url.locators.get('DISCOVER_APP_LOCATOR');
+    const job = mlJobService.getJob(this.props.anomaly.jobId);
+
+    const index = job.datafeed_config.indices[0];
+    const dataViewId = (await getDataViewIdFromName(index)) || index;
+
+    const record = this.props.anomaly.source;
+    const from = timeFormatter(record.timestamp); // e.g. 2016-02-08T16:00:00.000Z
+    const to = timeFormatter(record.timestamp + record.bucket_span * 1000);
+
+    let luceneQuery = '';
+
+    if (record.influencers) {
+      luceneQuery = record.influencers
+        .map((i) => `${i.influencer_field_name}:${i.influencer_field_values[0]}`)
+        .join(' AND ');
+    }
+
+    const url = await discoverLocator.getUrl({
+      indexPatternId: dataViewId,
+      refreshInterval: {
+        display: 'Off',
+        pause: true,
+        value: 0,
+      },
+      timeRange: {
+        from: from,
+        to: to,
+        mode: 'absolute',
+      },
+      query: {
+        language: 'lucene',
+        query: luceneQuery,
+      },
+      interval: 'auto',
+      sort: [['timestamp, asc']],
+    });
+
+    window.open(url);
```

Comment:
I think we should avoid using `window.open`. Shall we open in the same tab using `navigateToApp`?

Reply:
Is it possible to have the context menu item click open in a new tab using e.g. keyboard shortcuts?

Reply:
Updated to a href here https://github.com/elastic/kibana/pull/120450/commits/d6e4b9adae89b65416ff57ac59a43136b49e508b